src/api/helpers/map-CIS-countries/map-CIS-country/index.ts

```diff
@@ -24,14 +24,17 @@ export const mapCisCountry = (country: CisCountry): MappedCisCountry => {
   } as MappedCisCountry;
 
   mapped.canGetAQuoteOnline = canGetAQuoteOnline(mapped);
+  mapped.canGetAQuoteOffline = canApplyOffline(country.shortTermCoverAvailabilityDesc);
   mapped.canGetAQuoteByEmail = canGetAQuoteByEmail(mapped);
-
   mapped.cannotGetAQuote = cannotGetAQuote(mapped);
 
-  mapped.canApplyOnline = canApplyOnline(country.shortTermCoverAvailabilityDesc);
-  mapped.canApplyOffline = canApplyOffline(country.shortTermCoverAvailabilityDesc);
+  mapped.canApplyForInsuranceOnline = applyForInsuranceOnline(country.shortTermCoverAvailabilityDesc);
+  mapped.canApplyForInsuranceOffline = canApplyOffline(country.shortTermCoverAvailabilityDesc);
+
+  const noQuoteSupport = !mapped.canGetAQuoteOnline && !mapped.canGetAQuoteOffline;
+  const noInsuranceSupport = !mapped.canApplyForInsuranceOnline && !mapped.canApplyForInsuranceOffline;
 
-  mapped.cannotApply = !mapped.canApplyOnline && !mapped.canApplyOffline;
+  mapped.cannotGetAQuoteOrApplyForInsurance = !noQuoteSupport && !noInsuranceSupport;
```

Comment:
would this not be better as `noQuoteSupport && noInsuranceSupport`?

Reply:
Good shout - just had a look, we don't actually need this flag, this is only used for insurance/application elgibility. Will update

Reply:
Replaced this with `noInsuranceSupport`.